src/applications/vaos/new-appointment/redux/selectors.js

```diff
@@ -285,16 +279,15 @@ export function getFacilityPageV2Info(state) {
   } = newAppointment;
 
   const facilities = newAppointment.facilities[(typeOfCare?.id)];
-  const eligibilityStatus = getEligibilityStatus(state);
+  const eligibility = selectEligibility(state);
   const validFacilities = formInfo.schema?.properties.vaFacility.enum;
 
   return {
     ...formInfo,
     address: selectVAPResidentialAddress(state),
-    canScheduleAtChosenFacility:
-      eligibilityStatus.direct || eligibilityStatus.request,
+    canScheduleAtChosenFacility: eligibility?.direct || eligibility?.request,
```

Comment:
The returned object might be null now, before eligibility has been calculated. This is how most other data we cache in redux works, so figured it was worth doing it this way instead of always returning an empty object.